config/production.js

```diff
@@ -31,6 +31,7 @@ module.exports = {
     HOME: '/my-dashboard',
     COMMUNITY: 'https://community.topcoder.com',
     FORUMS: 'https://apps.topcoder.com/forums',
+    DISCUSSION_FORUM: 'https://discussions.topcoder.com',
```

Comment:
We already have a link to discuss forum in config: `FORUMS_VANILLA`

Reply:
@luizrrodrigues ok updating PR in sometime